tests/docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+networks:
+  redpanda:
+    name: redpanda
+    driver: bridge
+
+services:
+  n1: &node
+    image: vectorized/redpanda-test-node
+    privileged: true
+    volumes:
+    - '${BUILD_ROOT}:${BUILD_ROOT}'
+    networks:
+    - redpanda
+  n2: *node
+  n3: *node
+  n4: *node
```

Comment:
shouldn't this be tied to the concurrency in the ducktape tests - i thought we had a test w/ 6 things. 

if so i feel like we should have a max 'redpanda' container constant of 4 there in the python code somewhere.

Reply:
yeah, that'd be the next step. AFAIK we only run tests with 4 nodes, but yeah, the idea is to have this parametrized or having a template.

Reply:
starting the container is so light weight there is hardly any problem having 20 of them running. trying to automate this would be a lot hassle compared to once in a while bumping up the number?